api/src/main/java/org/openmrs/Person.java

```diff
@@ -54,7 +54,10 @@
 	private Set<PersonName> names = null;
 	
 	@ContainedIn
-	private Set<PersonAttribute> attributes = null;
+	@Override
+	public Set<PersonAttribute> getAttributes() {
```

Comment:
There's another version of `getAttributes()` starting around line 406 in this file. Just delete that.

Reply:
@ibacher I clear the entire method on line 406?

Reply:
Yes, that's the idea. We want to replace that with this implementation.